alibuild_helpers/deps.py

```diff
@@ -87,9 +87,9 @@ def doDeps(args, parser):
   dot += "}\n"
 
   if args.outdot:
-    fp = open(args.outdot, "w")
+    fp = open(args.outdot, "wt")
```

Comment:
why is this needed? Isn't "w" == "wt"?

Reply:
That's what I thought. [However](https://stackoverflow.com/questions/23051062/open-files-in-rt-and-wt-modes#comment35854536_23051114):

> In Python 3 there's an additional difference between text and binary file modes (on all platforms). In text mode, read returns Unicode strings. In binary mode, read returns a bytes instance. If you want to write Python 2 code with forwards compatibility in mind, you can use io.open rather than the standard open to get the Python 3 behavior (with unicode versus str instances).

which is our case precisely 😄